app/controllers/devise_token_auth/confirmations_controller.rb

```diff
@@ -21,12 +21,17 @@ def show
         yield @resource if block_given?
 
         redirect_to(@resource.build_auth_url(params[:redirect_url], {
-          "access-token":               token,
-          account_confirmation_success: true,
-          client:                       client_id,
-          client_id:                    client_id,
-          config:                       params[:config],
-          token:                        token
+          DeviseTokenAuth.headers_names["access-token"] => token,
+          DeviseTokenAuth.headers_names["client"] => client_id,
+
```

Comment:
Bit of a nitpick but any reason for an extra space here?

Reply:
I was attempting to make a distinction between the keys coming from `headers_names` and the bare keys - I don't feel strongly about the need for that, though.

Reply:
Ah not a bad idea. I'm fine with this then.